packages/api/endpoints/collections.js

```diff
@@ -138,11 +151,32 @@ async function del(req, res) {
   }
 }
 
+async function addToES(req, res) {
+  const collection = req.collectionRecord;
+
+  if (inTestMode()) {
+    const esClient = await Search.es('fakehost');
```

Comment:
So on this line and the next one, I think using env vars for the ES host and index name would be preferable. And doing this throughout the endpoint code

Reply:
To be more clear, I was thinking this would be

```
const esClient = await Search.es(process.env.esHost);
const esIndex = process.env.esIndex;
```